tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py

```diff
@@ -90,8 +90,8 @@ def test(self):
         leader_addr = self.nodes[LEADER].get_ip6_address(ADDRESS_TYPE.LINK_LOCAL)
 
         # Step 3 - Verify connectivity by instructing each device to send an ICMPv6 Echo Request to the DUT
-        self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD))
-        self.assertTrue(self.nodes[SSED_1].ping(leader_addr, timeout=2 * consts.CSL_DEFAULT_PERIOD))
+        self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD / 1000))
+        self.assertTrue(self.nodes[SSED_1].ping(leader_addr, timeout=2 * consts.CSL_DEFAULT_PERIOD / 1000))
```

Comment:
`CSL_DEFAULT_PERIOD` is defined as "units of ten symbols". Should we use `CSL_DEFAULT_PERIOD_IN_SECOND` instead?